packages/docusaurus-theme-classic/codeTranslations/ja.json

```diff
@@ -43,13 +43,13 @@
   "theme.docs.sidebar.responsiveCloseButtonLabel": "メニューを閉じる",
   "theme.docs.sidebar.responsiveOpenButtonLabel": "メニューを開く",
   "theme.docs.versions.latestVersionLinkLabel": "最新バージョン",
-  "theme.docs.versions.latestVersionSuggestionLabel": "最新のドキュメントは{latestVersionLink} ({versionLabel}) を見てください。",
+  "theme.docs.versions.latestVersionSuggestionLabel": "最新のドキュメントは{latestVersionLink} ({versionLabel})を見てください。",
   "theme.docs.versions.unmaintainedVersionLabel": "これは{siteTitle} {versionLabel}のドキュメントで現在はアクティブにメンテナンスされていません。",
-  "theme.docs.versions.unreleasedVersionLabel": "これはリリース前の{siteTitle} {versionLabel}のドキュメントです。.",
+  "theme.docs.versions.unreleasedVersionLabel": "これはリリース前の{siteTitle} {versionLabel}のドキュメントです。",
   "theme.lastUpdated.atDate": "{date}に",
   "theme.lastUpdated.byUser": "{user}が",
   "theme.lastUpdated.lastUpdatedAtBy": "{atDate}{byUser}最終更新",
-  "theme.tags.tagsListLabel": "タグ:",
+  "theme.tags.tagsListLabel": "タグ：",
```

Comment:
Do not need to change the ":" for appearance.

Reply:
does it have any special meaning in japanese?

Reply:
In the first place, there is nothing in Japanese that means ":". Generally this is just borrowed from English and has a similar meaning.